src/Uplink/Register.php

```diff
@@ -11,34 +11,34 @@ class Register {
 	 *
 	 * @since 1.0.0
 	 *
-	 * @param string $name Resource name.
 	 * @param string $slug Resource slug.
+	 * @param string $name Resource name
 	 * @param string $version Resource version.
 	 * @param string $path Resource path to bootstrap file.
 	 * @param string $class Resource class.
 	 * @param string $license_class Resource license class
 	 *
 	 * @return Resources\Resource
 	 */
-	public static function plugin( $name, $slug, $version, $path, $class, $license_class = null ) {
-		return Resources\Plugin::register( $name, $slug, $version, $path, $class, $license_class );
+	public static function plugin( $slug, $name, $version, $path, $class, $license_class = null ) {
```

Comment:
These variables were in the correct order, but they were named for the wrong things, so I swapped them.

If you follow the chain this creates, all the `register()` methods take a slug first and this matches the current documentation in the README.

e.g.

```php
Register::plugin(
	$plugin_slug,
	$plugin_name,
	$plugin_version,
	$plugin_path,
	$plugin_class,
	$license_class // This is optional.
);

Register::service(
	$service_slug,
	$service_name,
	$service_version,
	$plugin_path,
	$plugin_class
);
```